src/views/Account/index.tsx

```diff
@@ -1,17 +1,61 @@
+import { Outlet } from 'react-router-dom';
+import { useContext } from 'react';
+
 import Page from '#components/Page';
+import NavigationTabList from '#components/NavigationTabList';
+import NavigationTab from '#components/NavigationTab';
 import useTranslation from '#hooks/useTranslation';
+import UserContext from '#contexts/user';
+import routes from '#routes';
 
 import i18n from './i18n.json';
 
 // eslint-disable-next-line import/prefer-default-export
 export function Component() {
     const strings = useTranslation(i18n);
+    const { userDetails } = useContext(UserContext);
+    /*
+    const {
+        pending: mePending,
+        response: meResponse,
+    } = useRequest({
+        url: 'api/v2/user/me/',
+    });
+    */
```

Comment:
we can remove this, right?

Reply:
We can move this to other component, where it is needed, but this is just skeleton for now.